_posts/admin/2022-01-27-clusterproxy.md

```diff
@@ -157,6 +180,14 @@ location =/html5client/locale {
 }
 ```
 
+Add the following piece into the `location /bbb-01/html5client/locales` directive in `/usr/share/bigbluebutton/nginx/bbb-html5.nginx`:
```

Comment:
This should just go into the package.

Reply:
Hi! Sorry for the late. Could you please elaborate? I don't have much experience with the packages.

Reply:
Make a seperate commit to `bigbluebutton/build/packages-template/bbb-html5/bbb-html5.nginx` with these changes. They will be part of the packages in the next nightly build.